apps/rush-lib/src/scripts/install-run.ts

```diff
@@ -16,12 +16,15 @@ import * as childProcess from 'child_process';
 import * as fs from 'fs';
 import * as os from 'os';
 import * as path from 'path';
-import { IPackageJson } from '@microsoft/node-core-library';
+import {
+  IPackageJson,
+  FileConstants
```

Comment:
>FileConstants [](start = 2, length = 13)

install-run.ts cannot call `require('@microsoft/node-core-library')` because there is no node_modules folder